Unit tests panic when there is no redis running

_From @arschles on October 13, 2017 18:56_

If you run `go test -tags unit $$(glide nv)` on the host, with no redis host running, many tests fail as expected, but one panics:

```console
--- FAIL: TestGetExistingInstance (0.00s)
        Error Trace:    store_test.go:51
	Error:      	Expected nil, but got: &net.OpError{Op:"dial", Net:"tcp", Source:net.Addr(nil), Addr:net.Addr(nil), Err:(*net.DNSError)(0xc42007fd80)}
        Error Trace:    store_test.go:55
	Error:      	Should be true
        Error Trace:    store_test.go:56
	Error:      	Expected nil, but got: &net.OpError{Op:"dial", Net:"tcp", Source:net.Addr(nil), Addr:net.Addr(nil), Err:(*net.DNSError)(0xc42007ff00)}
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1294357]

goroutine 33 [running]:
testing.tRunner.func1(0xc4200811e0)
	/usr/local/opt/go/libexec/src/testing/testing.go:622 +0x29d
panic(0x12e0200, 0x14d2410)
	/usr/local/opt/go/libexec/src/runtime/panic.go:489 +0x2cf
github.com/Azure/azure-service-broker/pkg/storage.TestGetExistingInstance(0xc4200811e0)
	/Users/aaschles/gocode/src/github.com/Azure/azure-service-broker/pkg/storage/store_test.go:57 +0x257
testing.tRunner(0xc4200811e0, 0x134b1a8)
	/usr/local/opt/go/libexec/src/testing/testing.go:657 +0x96
created by testing.(*T).Run
	/usr/local/opt/go/libexec/src/testing/testing.go:697 +0x2ca
```

--- a/pkg/storage/store_test.go
+++ b/pkg/storage/store_test.go
@@ -123,7 +123,7 @@ func TestGetExistingBinding(t *testing.T) {
 	// Assert that the retrieval was successful
 	assert.True(t, ok)
 	assert.Nil(t, err)
-	// Assert that binding is not nil before using 
+	// Assert that binding is not nil before using
 	if assert.NotNil(t, binding, "binding should not be nil") {
 		assert.Equal(t, bindingID, binding.BindingID)
 	}

